go.mod

```diff
@@ -86,6 +86,7 @@ require (
 	github.com/nsqio/go-nsq v1.1.0
 	github.com/olivere/elastic/v7 v7.0.31
 	github.com/opencontainers/runc v1.0.3 // indirect
+	github.com/opensearch-project/opensearch-go v1.1.0
```

Comment:
Shouldn't we be using https://pkg.go.dev/github.com/opensearch-project/opensearch-go/v2 ?

Reply:
Thanks for the suggestion. I updated to v2

Reply:
Awesome, glad that worked! I'll have to look at the changes in more detail, but, on a quick note, I think you might need to add an import here as well: https://github.com/benthosdev/benthos/blob/main/public/components/all/package.go